docs/sources/mimir/configure/about-runtime-configuration.md

```diff
@@ -23,6 +23,8 @@ Runtime configuration values take precedence over command-line options.
 
 If multiple runtime configuration files are specified the runtime configuration files will be merged in a left to right order.
 
+If a runtime configuration file has the `.gz` extension, it will be decompressed before being read.
```

Comment:
```suggestion
If a runtime configuration file has the `.gz` extension, it's decompressed before being read.
```

Reply:
We try to avoid using "will" in docs and instead always opt for the present tense.

Reply:
Is there a way we can use active voice here without losing the meaning? If not, it's okay as-is.

Reply:
Okay I'll do this active voice:
```suggestion
Add a `.gz` extension to the configuration files that should be decompressed before parsing YAML.
```
Thank you. 

I'm going to merge it with this, but please don't hesitate to provide extra suggestions, I can update the docs in a separate PR later.